Tenduke.Client.WinFormsSample/Tenduke.Client.WinFormsSample.csproj

```diff
@@ -23,13 +23,13 @@
     </None>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Update="chromiumembeddedframework.runtime.win-arm64" Version="126.2.7" />
+    <PackageReference Update="chromiumembeddedframework.runtime.win-arm64" Version="135.0.17" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Update="chromiumembeddedframework.runtime.win-x64" Version="126.2.7" />
+    <PackageReference Update="chromiumembeddedframework.runtime.win-x64" Version="135.0.17" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Update="chromiumembeddedframework.runtime.win-x86" Version="126.2.7" />
+    <PackageReference Update="chromiumembeddedframework.runtime.win-x86" Version="135.0.17" />

```

Comment:
@hamish-smith I have used `update` for this reference, but later on I see the same dependency has the `include` which has a previous version, I had a read to https://stackoverflow.com/questions/71660693/what-is-the-difference-between-packagereference-update-and-packagereference-incl and it seems update will get to the latest and keep previous configurations, but I would like to have your toughts on this.

Reply:
I think it would be better to remove the `Update` entries and instead bump the version in the `Include` entries at ln 35, 36, and 37.